lsb_vsx/patches/host_VSX/05_install.patch

```diff
@@ -159,14 +159,191 @@
  # Set up the testroot 							#
  #									#
  #########################################################################
-@@ -625,37 +479,9 @@
+@@ -339,176 +193,6 @@
+ )
+ 
+ #########################################################################
+-#
+-# Create the TET/DTET config files
```

Comment:
Why are we removing so big part of this file? The comment in this patch is certainly worth adding. Is it because we use host_vsx only for building tests, not running them? I mean all of the changes in this patch, not only the previously added - short comment explaining that should be left.